tests/python/frontend/pytorch/test_forward.py

```diff
@@ -5432,6 +5430,46 @@ def forward(self, *args):
     verify_model(Swapaxes3().float().eval(), input_data=input_data)
 
 
+def test_linalg_vector_norm():
+    """test_linalg_vector_norm"""
+    torch.set_grad_enabled(False)
+    input_shape = [3, 3]
+
+    class VectorNorm1(torch.nn.Module):
+        def forward(self, x):
+            return torch.linalg.vector_norm(x)
+
+    class VectorNorm2(torch.nn.Module):
+        def forward(self, x):
+            return torch.linalg.vector_norm(x, ord=3.5)
+
+    class VectorNorm3(torch.nn.Module):
+        def forward(self, x):
+            return torch.linalg.vector_norm(x, ord=np.inf)
+
+    class VectorNorm4(torch.nn.Module):
+        def forward(self, x):
+            return torch.linalg.vector_norm(x, ord=-np.NINF)
+
+    class VectorNorm5(torch.nn.Module):
+        def forward(self, x):
+            return torch.linalg.vector_norm(x, ord=0)
```

Comment:
if you only need different `ord` for each test case, you don't need to create a different class for each. Please clean them up.

Reply:
@masahi Revised as suggested. Please review. Thanks!